screwdriver/api/api.go

```diff
@@ -98,10 +98,10 @@ func handleResponse(res *http.Response) ([]byte, error) {
 }
 
 // GetCommand returns Command from Screwdriver API
-func (c client) GetCommand(namespace, command, version string) (*Command, error) {
+func (c client) GetCommand(namespace, name, version string) (*Command, error) {
 	cmd := new(Command)
 
-	url := fmt.Sprintf("%scommands/%s/%s", c.baseURL, namespace+"%2F"+command, version)
+	url := fmt.Sprintf("%scommands/%s/%s", c.baseURL, namespace+"%2F"+name, version)
```

Comment:
`namespace+"/"+name` is a valid URL for now. Please modify it.
https://github.com/screwdriver-cd/screwdriver/pull/838/files#diff-8aa94dca0af8f40f7dff7f86d35a8bd3R14

Reply:
I see. I will fix this.